### PR TITLE
fix: fix filterable dropdown style bug and json schema form bug

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] Updated Storybook documents for component
- [ ] Wrote test codes for composable, utils and etc.
- [x] Tested with console(if usages are changed) 

### Description

Commit messages say it all

### CODE REVIEW

File: src/inputs/forms/json-schema-form/helper.ts [hunk 1]
```diff
@@ -77,9 +77,13 @@ export const initFormDataWithSchema = (schema?: JsonSchema, formData?: object):
...
-                const isSearchDropdownType = !!getMenuItemsFromSchema(property);
-                const keyProperty = isSearchDropdownType ? 'name' : 'value'; // 'name' for PFilterableDropdown, 'value' for PTextInput
-                result[key] = result[key].map((d) => ({ [keyProperty]: d }));
+                const menuMap = {};
+                if (Array.isArray(property.menuItems)) {
+                    property.menuItems.forEach((d) => {
+                        if (d.name) menuMap[d.name] = d;
+                    });
+                }
+                result[key] = result[key].map((d) => ({ name: d, label: menuMap[d]?.label ?? d }));
```
Comment (yuda110) [inline src/inputs/forms/json-schema-form/helper.ts:86]: praise: Thank you so much🥹

**Review by yuda110 — APPROVED** [review]
LGTM!!!!!

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**